components/automate-ui/src/app/page-components/deletable-node-control/deletable-node-control.component.html

```diff
@@ -16,7 +16,7 @@ <h2 id="confirm-delete-modal-label" class="status-modal-heading">Warning</h2>
       <strong>{{nodesSelectedForDelete.size}} node{{ nodesSelectedForDelete.size | i18nPlural: pluralMapping }}</strong>.
     </p>
     <p>This action can not be undone.</p>
-    <chef-button primary caution (click)="closeModal(true)">
+    <chef-button class="delete-button-confirm" primary caution (click)="closeModal(true)">
```

Comment:
Added to allow the cypress test to find this control.

Reply:
In other places, I believe we've used data-cy for that.

Reply:
ya, https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements